app/src/androidTest/java/com/github/se/project/ui/lesson/TutorMatchingTest.kt

```diff
@@ -62,7 +62,7 @@ import androidx.compose.ui.test.*
                   academicLevel = AcademicLevel.MA1,
                   languages = listOf(Language.ENGLISH),
                   subjects = listOf(Subject.ANALYSIS),
-                  schedule = listOf(),
+                  schedule = List(7) { List(12) { 1 } },
```

Comment:
Thank you for catching these bugs in the test! I don't even know how it passed before.